### DISCUSSION TITLE
Firebase Integration Example

### DISCUSSION THREAD

**Discussion by gonesurfing** [vikejs/vike#1254] [Discussion] [answered]
FYI, if anyone is trying to get this to work with Firebase hosting, I have an example repo below.

https://github.com/gonesurfing/vike-ssr

I can do a PR to get this into the V1 docs. I thought it might be useful here in case someone is searching.

**Comment by brillout** [comment]
That's great and, yes, a docs PR would be wondeful. Looking forward to it 🙂

**Reply by brillout** [reply, marked as answer]
I just added https://github.com/gonesurfing/vike-ssr to https://vike.dev/firebase.

**Comment by gonesurfing** [comment]
Cool.  I will still submit a PR so that it can live with the Vike project docs long term.

**Reply by brillout** [reply]
I'm not sure what you mean: it's already listed at https://vike.dev/firebase :)

**Reply by gonesurfing** [reply]
Should the whole example be in "/vike/examples/vike-firebase-functions" or something similar, instead of living in my repo?

**Reply by brillout** [reply]
Depends whether 1. there is a higher likelihood Vike introduces a breaking change regarding deploying to Firebase and 2. there is a way to test the Firebase deployment locally (e.g. Cloudflare's wrangler can preview deployments) so that Vike's CI can test it.

If yes then it'd indeed be nice to make it an official example and ensure Vike doesn't break Firebase deployment.

**Reply by gonesurfing** [reply]
Gotcha. 
I'm not sure of the likelihood of 1., but for 2. there is a way to simulate Firebase Hosting/Functions locally. We can table it for now, and if I ever develop a CI test, we can add it to the repo then. Thanks.

**Reply by brillout** [reply]
> for 2. there is a way to simulate Firebase Hosting/Functions locall

Very neat. That's a strong argument to add it to the CI. PR welcome 👍
